### DISCUSSION TITLE
Export svg format picture text garbled

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#379] [Q&A]
Hello! Dear Mr. Mozman:
I would like to ask you the following. When using ezdxf backend to convert dxf files to svg image format, garbled characters will be generated. Is there any way to solve it?
such as：
![image](https://user-images.githubusercontent.com/59160360/112783967-11485280-9083-11eb-93f0-b9e9ff8a8d9b.png)

**Comment by luoyu-123** [comment]
Supplementary note：
There is a warning notice: as shown below
![image](https://user-images.githubusercontent.com/59160360/112807567-a448b300-90aa-11eb-84d2-79f6946381a0.png)
Is there any way to solve this problem？

thanks

**Comment by mbway** [comment]
it looks like you don't have the font that the document uses installed, and the fallback font (DejaVu Sans) does not have the glyphs for the language(s) used in the document. You could try using a different fallback font with better coverage or install the font that the document is looking for. Droid sans seems to be a popular option and has support for many languages: https://en.wikipedia.org/wiki/Droid_fonts#Typefaces
